app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,78 @@
+class RecipesController < ApplicationController
```

Comment:
- [ ] Kindly write unit and integration tests as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Write%20unit%20and%20integration%20tests).

Reply:
thanks, I have fixed it.